code/modules/mob/living/carbon/xenomorph/castes/King.dm

```diff
@@ -0,0 +1,107 @@
+/datum/caste_datum/king
+	caste_type = XENO_CASTE_KING
+	caste_desc = "The end of the line."
+	tier = 4
+
+	melee_damage_lower = XENO_DAMAGE_TIER_6
+	melee_damage_upper = XENO_DAMAGE_TIER_8
+	melee_vehicle_damage = XENO_DAMAGE_TIER_5
+	max_health = XENO_HEALTH_KING
+	plasma_gain = XENO_PLASMA_GAIN_TIER_3
+	plasma_max = XENO_PLASMA_TIER_10
+	xeno_explosion_resistance = XENO_EXPLOSIVE_ARMOR_TIER_7
+	armor_deflection = XENO_ARMOR_FACTOR_TIER_5
+	speed = XENO_SPEED_TIER_1
+
+	evolves_to = null
+	deevolves_to = null
+	evolution_allowed = FALSE
+	can_vent_crawl = FALSE
+
+	behavior_delegate_type = /datum/behavior_delegate/king_base
+
+	tackle_min = 6
+	tackle_max = 10
+
+	minimap_icon = "xenoqueen"
+
+	fire_immunity = FIRE_IMMUNITY_NO_DAMAGE
+
+/datum/caste_datum/king/New()
+	minimum_evolve_time = XENO_KING_ACQUISITION_TIME
+
+	return ..()
+	
```

Comment:
```suggestion
```

Not only is the define giving currently a non-constant value currently, it is far too late to be assigning it for how I was using it. See `/proc/setup_xeno_evolve_times()` That proc would only ever read from what the caste has defined.

So what you need to do instead is just not specify this caste's `minimum_evolve_time` anymore, and manually alter GLOB.xeno_evolve_times when you set up the glob for the time random selector (see previous review comment).